README.md

```diff
@@ -440,10 +441,11 @@ The device plugin will initially discover all PCI network resources in the host
 2. "devices"      - The device hex code of device
 3. "drivers"      - The driver name the device is registered with
 4. "pciAddresses" - The pci address of the device in BDF notation (if device type is accelerator or netDevice)
-5. "auxTypes"     - The type of auxiliary network device (if device type is auxNetDevice).
-6. "pfNames"      - The Physical function name
-7. "rootDevices"  - The Physical function PCI address
-8. "linkTypes"    - The link type of the net device associated with the PCI device
+5. "acpiIndexes"  - The acpi index of the device
```

Comment:
it seems to me we need a bit more info about acpi index, when its available etc...